ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldFEM.h

```diff
@@ -12,7 +12,9 @@
 #include <memory>
 #include <vector>
 
-#include "MaterialLib/SolidModels/PhaseFieldExtension.h"
+#include "MaterialLib/SolidModels/LinearElasticIsotropic.h"
```

Comment:
This include is not needed because it is included  in /LinearElasticIsotropic.h too.

Reply:
The `LinearElasticIsotropicPhaseField.h` does not include the `LinearElasticIsotropic.h` any longer. The bulk modulus etc. is passed to the function directly.